library/std/src/f16/tests.rs

```diff
@@ -0,0 +1,46 @@
+#![allow(dead_code)] // FIXME(f16_f128): remove once constants are used
```

Comment:
I presume this is just missing tests that actually use these constants? Will that come in a later step?

Reply:
That is correct, I am planning to add the constants and tests at the same time.